smrf/data/load_topo.py

```diff
@@ -91,7 +90,14 @@ def readNetCDF(self):
         self._logger.info('Domain center as Latitude/Longitude = {:0.5f}, '
                           '{:0.5f}'.format(self.basin_lat, self.basin_long))
 
-        f.close()
+        # Load or calculate the sky view factor
+        if 'sky_view_factor' in f.variables:
+            self.sky_view_factor = f['sky_view_factor'][:]
+            self.terrain_config_factor = f['terrain_config_factor'][:]
+            f.close()
+        else:
+            f.close()
+            self.calculate_sky_view_factor()
```

Comment:
This was an added improvement, storing the sky_view_factor as variables on the topo instead of calculating it with each model execution. The factors are constant and only depend on topography.